forms-flow-web/src/helper/formUtils.js

```diff
@@ -27,4 +27,21 @@ const scrollToErrorOnValidation = (formio, scrollToErrorInterval) => {
   }
 };
 
-export { convertFormLinksToOpenInNewTabs, scrollToErrorOnValidation };
+const enableFormButton = (formio, enableButtonInterval, buttonKey, onClick) => {
```

Comment:
the usual emit event doesn't work on the `submission.view` since the form is `readOnly`. As a workaround wrote this helper.